scripts/o3de/o3de/sha256.py

```diff
@@ -35,7 +35,7 @@ def sha256(file_path: str or pathlib.Path,
             logger.error(f'Json path {json_path} does not exist.')
             return 1
 
-    the_sha256 = hashlib.sha256(file_path.open('rb').read()).hexdigest()
+    sha256 = hashlib.sha256(file_path.open('rb').read()).hexdigest()
```

Comment:
is it good practice to have a local variable with the same name as the function?

Reply:
sha256 and hash.sha256 are not the same name right?

Reply:
Just to be safe I changed it back

Reply:
minor point: technically, it should be 'sha_digest' or 'sha256_digest' since sha256 is the name of the algorithm, not its output